### PR TITLE
Fixed #27931 -- Add explanation for clarifing  meaning of django catch-all logger

### PR DESCRIPTION
**Pull request by nerogit**
https://code.djangoproject.com/ticket/27931

### CODE REVIEW

File: docs/topics/logging.txt [hunk 1]
```diff
@@ -734,8 +734,10 @@ By default, Django configures the following logging:
 
 When :setting:`DEBUG` is ``True``:
 
-* The ``django`` catch-all logger sends all messages at the ``INFO`` level or
-  higher to the console.
+* The ``django`` catch-all logger sends messages at the ``INFO`` level or
+  higher to the console. Obviously, the logger does not contain other other package's messages, but only
```
Comment (Kangaroux) [inline docs/topics/logging.txt]: I'd change "Obviously" to "Of course".

**Comment by darjeeling** [comment]
It looks like djangoci failed to push junit reports.
can anyone run ci again?

File: docs/topics/logging.txt [hunk 2]
```diff
@@ -734,8 +734,10 @@ By default, Django configures the following logging:
 
 When :setting:`DEBUG` is ``True``:
 
-* The ``django`` catch-all logger sends all messages at the ``INFO`` level or
-  higher to the console.
+* The ``django`` catch-all logger sends messages at the ``INFO`` level or
+  higher to the console. Of course, the logger does not contain the other package's messages, but only
+  processes messages in the ``django`` namespace and also it's sub namespaces, such as ``django.request``,
```
Comment (vlt) [inline docs/topics/logging.txt]: s/it's/its

File: docs/topics/logging.txt [hunk 3]
```diff
@@ -734,8 +734,10 @@ By default, Django configures the following logging:
 
 When :setting:`DEBUG` is ``True``:
 
-* The ``django`` catch-all logger sends all messages at the ``INFO`` level or
-  higher to the console.
+* The ``django`` catch-all logger sends messages at the ``INFO`` level or
+  higher to the console. Of course, the logger does not contain the other package's messages, but only
```
Comment (vlt) [inline docs/topics/logging.txt]: s/package's/packages'

**Comment by nerogit** [comment]
@vlt I fixed it. Thanks :)

**Comment by timgraham** [comment]
I pushed an alternate proposal here. I used the term "hierarchy" rather than "namespace" to match the Python docs. Let me know what you think.

**Comment by Kangaroux** [comment]
I personally found the previous copy to be easier to understand.

**Comment by nerogit** [comment]
@timgraham Seems to be getting better. But I think 

> The django logger sends `all` messages at the \`\`INFO\`\` level or higher to the console.

this is more explicit, as previous version.

**Comment by timgraham** [comment]
I wasn't sure if "all" might be a source of confusion for the ticket's reporter. The logger doesn't handle "all" messages in the sense that it only handles messages in its name hierarchy. To me, the sentence's meaning doesn't really change whether or not "all" is there. If we agree to remove it, then I noticed there's another "all" where the django-server logger is discussed a few lines later that could also be removed.

**Comment by timgraham** [comment]
@nerogit, what do you think about my comment?

**Comment by nerogit** [comment]
I wanted to write a because of the consistency of sentences. Let's remove "all" in django-server logger

**Comment by timgraham** [comment]
Done. And I pushed another amendment to hopefully address @Kangaroux's concerns about the initial wording being easier to understand. Let me know!

**Comment by Kangaroux** [comment]
Looks good 👍

**Pull request merged**